docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_map_expectations.md

```diff
@@ -1,31 +1,249 @@
+
+```bash
+mv column_map_expectation_template.py /SOME_DIRECTORY/expect_column_values_to_equal_three.py
+```
 
-#### 2. Implement your Metric
+<details>
+  <summary>Where should I put my Expectation file?</summary>
+  <div>
+    <p>
+        During development, you don't actually need to put the file anywhere in particular. It's self-contained, and can be executed anywhere as long as <code>great_expectations</code> is installed.
+    </p>
+    <p>
+        But to use your new Expectation alongside the other components of Great Expectations, you'll need to make sure the file is in the right place. The right place depends on what you intend to use it for.
+    </p>
+    <p>
+        <ul>
+            <li>If you're building a Custom Expectation for personal use, you'll need to put it in the <code>great_expectations/plugins/expectations</code> folder of your Great Expectations deployment. When you instantiate the corresponding <code>DataContext</code>, it will automatically make all plugins in the directory available for use.</li>
+            <li>If you're building a Custom Expectation to contribute to the open source project, you'll need to put it in the repo for the Great Expectations library itself. Most likely, this will be within a package within <code>contrib/</code>: <code>great_expectations/contrib/SOME_PACKAGE/SOME_PACKAGE/expectations/</code>. To use these Expectations, you'll need to install the package.</li>
+        </ul>
+    </p>
+  </div>
+</details>
+
+#### 3. Run checklist diagnostics on your file
+
+Once you've copied and renamed the template file, you can execute it as follows.
+
+```bash
+python expect_column_values_to_equal_three.py
+```
+
+The template file is set up so that this will run the Expectation's `generate_diagnostic_checklist` method. This will run a diagnostic script on your new Expectation, and return a checklist of steps to get it to full production readiness.
+
+```
+Completeness checklist for ExpectColumnValuesToMatchSomeCriteria:
+  ✔ library_metadata object exists
+    Has a docstring, including a one-line short description
+    Has at least one positive and negative example case
+    Core logic exists and passes tests on at least one Execution Engine
+    Has all four statement Renderers: question, descriptive, prescriptive, diagnostic
+    Has default ParameterBuilders and Domain hooks to support Profiling
+    Core logic exists and passes tests for all applicable Execution Engines and backends
+    All Renderers exist and produce typed output
+    Linting for type hints and other code standards passes
+    Input validation exists
+```
+
+When in doubt, the next step to implement is the first one that doesn't have a ✔ next to it. This guide covers the first four steps on the checklist.
+
+#### 4. Change the Expectation class name and add a docstring
+
+Let's start by updating your Expectations's name and docstring.
+
+Replace the Expectation class name
+```python
```

Comment:
Does this code exist in the repo? Can it be referenced here and put under test? Example from "how_to_configure_a_configuredassetdataconnector.md"
<pre>
```python file=../../../tests/integration/docusaurus/connecting_to_your_data/how_to_configure_a_configuredassetdataconnector.py#L1-L4
```
</pre>

Reply:
@donaldheppner This code now exists in the repo, references have been updated as above.